polkadot-parachains/encointer-runtime/src/lib.rs

```diff
@@ -414,6 +418,7 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	>;
 	type ControllerOriginConverter = XcmOriginToTransactDispatchOrigin;
 	type WeightInfo = weights::cumulus_pallet_xcmp_queue::WeightInfo<Runtime>;
+	type PriceForSiblingDelivery = ();
```

Comment:
Did you double check it's significance? With all newly added entries, I want that either or both:

* We have fully understood it's meaning and we know that the value we put is fine
* We have crosschecked the value with what is on Statemin* or Polkadot/Kusama

Reply:
Beautiful. Thanks! From now on, I am wary about `()` :)